docs/howto/static-files/index.txt

```diff
@@ -46,7 +46,7 @@ Configuring static files
     See :doc:`/howto/static-files/deployment` for proper strategies to serve
     static files in production environments.
 
-Your project will probably also have static assets that aren't tied to a
+Your project will probably also have static assets that are not tied to a
```

Comment:
Thanks, but the original is correct. The word `aren't` is a contraction of `are not` and I don't think it's worth changing.